### DISCUSSION TITLE
Opening calendar inside of modal is getting cut off

### DISCUSSION THREAD

**Discussion by jonathan1228** [mantinedev/mantine#4257] [Q&A] [answered]
Is there a way to display the Calendar without it being cut off due to the height of a modal? I don't want to increase the height and using that as a last resort but I've tried CSS and can't seem to get the calendar to display fully inside of a modal

**Comment by rtivital** [comment, marked as answer]
Set `<DatePickerInput popoverProps={{ withinPortal: true }} />`
